les/winnercheque-v1.html

```diff
@@ -11,13 +11,17 @@ <h4 id="chequeId"><a href="#chequeId">chequeId</a></h4>
 
 Unique identifier belonging to the cheque.
 
-<h4 id="depositAddress"><a href="#depositAddress">depositAddress</a></h4>
+<h4 id="cryptoAddress"><a href="#cryptoAddress">cryptoAddress</a></h4>
 
-The wallet address of the recipient.
+The crypto address of the recipient.
 
 <h4 id="monetaryAmount"><a href="#monetaryAmount">monetaryAmount</a></h4>
 
-The amount that should be transferred to the wallet.
+The amount that should be transferred to the crypto address.
+
+<h4 id="monetaryCurrency"><a href="#monetaryCurrency">monetaryCurrency</a></h4>
+
+The currency of the amount that should be transferred to the wallet.
```

Comment:
wallet or crypto address, as this is what we call it everywhere

Reply:
The currency type that should be transferred to the wallet. ?

Reply:
Just address

Reply:
monetaryCurrency will be "USDC"